foreman_discovery.gemspec

```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |s|
   s.summary = %q{MaaS Discovery Plugin for Foreman}
 
   s.add_dependency 'deface', '< 2.0'
+  s.add_dependency 'rails', '>= 4.1'
```

Comment:
this will allow versioning of the plugin in a way that bundler can decide which version of this to use. in case of older codebase, bundler will not use this plugin version because of the dependency. I think it should stay.